clients/vscode/package.json

```diff
@@ -143,7 +143,7 @@
       {
         "title": "Tabby",
         "properties": {
-          "tabby.api.endpoint": {
+          "tabby.API.endpoint": {
```

Comment:
It seems that this requires changes in code where read and write this setting.

Reply:
removed this change